src/main/java/com/github/nutomic/controldlna/gui/RouteFragment.java

```diff
@@ -671,6 +671,7 @@ public void receivePlaybackStatus(MediaItemStatus status) {
 
 		mProgressBar.setProgress(currentTime);
 		mProgressBar.setMax(totalTime);
+		mProgressBar.setKeyProgressIncrement(180);
```

Comment:
Would it make more sense for this to be 5% of total duration rather than a fixed time - for a song that is only around 3 minutes long 180 seconds is too long. On the other hand there aren't many devices around that still support dpad to take advantage of it.

Reply:
Good point, I changed the progress bar skip increment to 5% of song length.